app_flutter/web/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
You need one here in an html comment, see https://github.com/flutter/flutter/blob/master/examples/hello_world/web/index.html#L1

Reply:
Added! Thank you!